variables.tf

```diff
@@ -292,6 +292,8 @@ variable "create_database_nat_gateway_route" {
   default     = false
 }
 
+
+
 variable "firewall_sync_states" {
```

Comment:
with `enable_firewall` and creation of firewall resource in the module, this variable is no longer needed